deploy_development.env

```diff
@@ -0,0 +1,15 @@
+#### #### #### ####
+# Encrypted Values
+#### #### #### ####
+
+CARD_CREATOR_USERNAME=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHwwegYJKoZIhvcNAQcGoG0wawIBADBmBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDOwV7ZC8HGOJVmdzQgIBEIA5zcd1yYupstx/5f++p9g0GyxZc4Jw1/l3+mGa1ouTb1LYGyCaaO6DuLm0gfS0PZ/CZEmFWHKq4WW/
```

Comment:
One thing I wanted to add:

Currently, I believe, there is no decryption functionality in the app. So, obviously, this will not work when deployed to development. We probably should create a separate task/tasks for this work. Just as a heads-up, there are other Node apps that decrypt environment variable so you might want to look to them as examples.

Reply:
Sounds good @kfriedman I added it here: https://jira.nypl.org/browse/MLN-47

Reply:
@kfriedman I've merged in the [feature branch related to AWS KMS](https://github.com/NYPL/dgx-patron-creator-service/pull/2).  The integration test is successfully interacting with the QA instance of Card Creator and the Patron Kinesis stream in the Sandbox environment.  Does that address the concern you raised in the comment above "no decryption functionality in the app"?

Reply:
This is great @LucasCioffi ! Thanks for your work on this.